src/main/java/org/example/Main.java

```diff
@@ -1,5 +1,52 @@
-package org.example;public class Main {
+package org.example;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/*
+1. Создать программу для сохранения заказа на покупку товара в файл.
+Данные о заказе (наименование товара, количество, общая стоимость)
+вводятся пользователем с клавиатуры.
+Сделать логироване действий пользователя в файл.
+Предусмотреть различные уровни логирования: общая информация,
+предупреждающее сообщения, сообщения об ошибках.
+
+2. Для предыдущего задания создать unit-тесты. Предусмотреть покрытие
+кода тестами минимум на 50%.
+ */
+public class Main {
+
+    private static final Logger log =Logger.getLogger(String.valueOf(Main.class));
+
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        try {
+            Scanner in = new Scanner(System.in);
```

Comment:
Your program should allow to process several orders, untill you'll quit the application

Reply:
How about this note?